frontend/src/data/nsf313_30/query.spec.ts

```diff
@@ -0,0 +1,145 @@
+import { describe, expect, test } from '@jest/globals';
+import { Query_nsf313_30 } from './query'
+import { Query } from '../queries/query'
+import { Categories } from './categories';
+
+const demo = new Query_nsf313_30()
+
+const black = new Query('race', 'Black')
+const four_year = new Query('collegeOrUniversityLevel',
+    '4year',)
+
+const private_enrollment = new Query('collegeOrGraduateSchoolEnrollment', 'EnrolledInPrivateCollegeOrGraduateSchool')
+const female = new Query('gender', 'Female')
+
+const all_genders = Categories['gender']
+
+const all_enrollment_types = Categories['collegeOrGraduateSchoolEnrollment']
+
+const all_university_levels = Categories['collegeOrUniversityLevel']
+
+describe('demo query', () => {
+
+    test('test race alone', () => {
+        let out = demo.query(black)
+        expect(out.error).toEqual(undefined)
+        expect(out.results).toEqual([
+            "Count_Person_EnrolledInPublicCollegeOrGraduateSchool_Female_HBCU_Black",
+            "Count_Person_EnrolledInPublicCollegeOrGraduateSchool_Male_HBCU_Black",
+            "Count_Person_EnrolledInPrivateCollegeOrGraduateSchool_Female_HBCU_Black",
+            "Count_Person_EnrolledInPrivateCollegeOrGraduateSchool_Male_HBCU_Black",
+        ])
+    })
+
+    test('test race enrollment', () => {
+        let out = demo.query(black, private_enrollment)
+        expect(out.error).toEqual(undefined)
+        expect(out.results).toEqual([
+            "Count_Person_EnrolledInPrivateCollegeOrGraduateSchool_Female_HBCU_Black",
+            "Count_Person_EnrolledInPrivateCollegeOrGraduateSchool_Male_HBCU_Black",])
+    })
+
+    test('test gender alone', () => {
```

Comment:
see my comment about gender not being dependent on enrollment. If we remove that dependency, we'd return the single key here:

https://github.com/Spelman-College/spelman-dashboard/pull/87/files#diff-6ceefa38fba06ad66817dc6e8d4a6ea41126735addeccc5653db2397f35672fbR24

Reply:
done